lib/gui/app.js

```diff
@@ -85,23 +86,44 @@ app.run(() => {
 app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateModel) => {
   AnalyticsService.logEvent('Application start');
 
-  if (UpdateNotifierService.shouldCheckForUpdates() && !process.env.ETCHER_DISABLE_UPDATES) {
-    AnalyticsService.logEvent('Checking for updates');
+  const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates();
 
-    UpdateNotifierService.isLatestVersion().then((isLatestVersion) => {
+  if (!shouldCheckForUpdates || process.env.ETCHER_DISABLE_UPDATES) {
+    AnalyticsService.logEvent('Not checking for updates', {
+      shouldCheckForUpdates,
+      ETCHER_DISABLE_UPDATES: process.env.ETCHER_DISABLE_UPDATES
```

Comment:
The mixpanel context transformation logic is causing the underscores to get removed:

```
Wed Feb 22 2017 16:51:49 GMT-0400 (BOT) Not checking for updates ({"Should Check For Updates":true,"ETCHER DISABLE UPDATES":"true"})
```

We should either:

- Make the transformation keep things like environment variable names intact, or
- Describe this property in a way that is not tied to the actual environment variable name, like "updatesDisabled"

I think I prefer the second option.